Platform/ArrowlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -801,6 +802,7 @@ UpdateFspConfig (
       FspsConfig->PcieGen4EqPh1UpTxPreset[2] = 0x7;
       FspsConfig->PcieGen4EqPh1UpTxPreset[4] = 0x7;
       FspsConfig->Usb4CmMode                 = 0x0;
+      StoreRpConfig (FspsConfig);
```

Comment:
@kokweich find the design in Adl code, this function call is invoked only PcdEnablePciePm set to True. simliar to PciePmConfig function in Stage2BoardInitLic. Suggest you make code align with other project. thanks

Reply:
Added the PcdEnablePciePm to both fspsconfig and stage2boardinitlib to only invoke the function when PcdEnablePciePm is set to true